test/verify/check-system-services

```diff
@@ -221,8 +221,10 @@ Unit=test.service
         # Make sure that symlinked services also appear in the list
         self.wait_service_state("reboot.target", "inactive")
         self.wait_service_state("ctrl-alt-del.target", "inactive")
-        self.wait_service_state("runlevel6.target", "inactive")
-        self.goto_service("runlevel6.target")
+        # Arch Linux has no SYSV_COMPAT runlevels https://github.com/systemd/systemd/blob/eaaa02bffe3b49504b58fad1980ac22eef3002cd/units/meson.build#L3
+        runlevel = "reboot.target" if m.image == "arch" else "runlevel6.target"
+        self.wait_service_state(runlevel, "inactive")
+        self.goto_service(runlevel)
```

Comment:
Incidentally, @KKoukiou just cleaned that up in PR #16545 . Funny how things play out, I said "let's remove it, although it won't get in the way anytime soon".. and here we are!

Reply:
Cool, I rebased and dropped it.